src/components/CartList/CartList.js

```diff
@@ -0,0 +1,70 @@
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import { Link } from "react-router-dom";
+
+
+const  CartList=({product})=>{
+      const {_id,name,categoryName,price,discountedPrice,discount,cardImage,quantity} =product;
+      const {cartDispatch}=useCart(); 
+      const {wishlistState,wishlistDispatch}=useWishlist();
+      
+    return (
+        
+             <div className="cart-products">
+            <div className="card5 horizontal-card-container">
+                <div className="card-img">
+                    <img className="horizontal-card-img" src={cardImage} alt="ecomimage" />
+                </div>
+                <div className="card-info">
+
+                    <h2 className="card-title">{name}</h2>
+                    <h3 className="subtitle">{categoryName}</h3>
+                    <div className="card-body">
+                        <div className="price-details">
+                            <p className="disc-price"> {discountedPrice}</p>
+                            <p className="actual-price">{price}</p>
+                            <p className="offer-info">({discount}% Off)</p>
+
+
+                        </div>
+                    </div>
+                    <div className="product-quantity">
+                        <p>Quantity:</p>
+                        <button className="btn minus-quantity" 
+                        onClick={()=>cartDispatch({type:"Decrease_Quantity",payload: _id})}><i className="material-icons">remove</i></button>
```

Comment:
Could , in later versions, some functionality be written so that If the quantity of the product is one, then the on click fires this function ? onClick={()=>cartDispatch({type:"RemoveFromCart", payload:_id})}

As the minus button does nothing, if the quantity is one ,just a finer things suggestion, may get you extra points maybe :)